front-end/src/components/player-page/stats/ping-overview.tsx

```diff
@@ -0,0 +1,87 @@
+import {Box, Fade, Theme, Tooltip, Typography} from "@mui/material";
+import React from "react";
+import BaseRisenBox from "../../risen-box/base-risen-box";
+import PlayerStatModel from "../../../../../Common/models/playerstat.model";
+import {useTheme} from "@emotion/react";
+import {StatGenerators} from "../../../common/constants";
+import {BaseStatGenerator} from "../../../common/stats-generators/BaseStatsGenerator";
+import {playerStatsHasData} from "../../../../../Common/utils";
+import {abbreviateNumber} from "../../../common/utils";
+
+interface PingOverviewProps {
+    playerStats: PlayerStatModel[]
+}
+
+export const pings: Record<string, BaseStatGenerator> = {
```

Comment:
Don't think this is required to be exported

Reply:
Fixed in rev2

Reply:
Tf is a rev? I see no revs on this site